Makefile

```diff
@@ -3,11 +3,13 @@
 
 PREFIX = /usr/local
 LIBDIR = ${PREFIX}/lib
+LIBEXECDIR = ${PREFIX}/libexec
 INCLUDEDIR = ${PREFIX}/include
 PKGCONFIGDIR = ${LIBDIR}/pkgconfig
 XCFLAGS = ${CPPFLAGS} ${CFLAGS} -std=c99 -fPIC -D_XOPEN_SOURCE=700 \
 		  -Wall -Wextra -Wpedantic -Wmissing-prototypes -Wstrict-prototypes \
-		  -Wno-unused-parameter
+		  -Wno-unused-parameter \
+		  -DRULES_HELPER_PATH=\"${LIBEXECDIR}/libudev-zero-rules-helper\"
```

Comment:
It'd be better if RULES_HELPER_PATH had its own make macro, so user could set it on the command line when building just like we set PREFIX and similar...